dist/main.js

```diff
@@ -0,0 +1,65 @@
+let currTeam = []
+let dreamTeam = []
+$("#roster").on("click", function() {
+    let teamName = $("input").val()
+    $.get(`/teams/${teamName}`, function(theTeam){
+        renderTeam(theTeam)
+        currTeam = theTeam
+    })
+})
+
+$("#content").on("click", "#notDreamTm", function() {
+    let playerId = $(this).closest(".card").data().id
+    let player = currTeam.find(p => (p.pId == playerId))
+    if(!player.isDream){
+        player.isDream = true
+        addToDreamTeam(player)
+    }
+    renderTeam(currTeam)
```

Comment:
Why rendering in case _player.isDream_ is true?
Don't you need to render it inside the if statement